### ISSUE TITLE
Except on filesize mismatch

### ISSUE THREAD

**Issue by Xalaxis** [andrey-yantsen/plexiglas#7]
Instead of raising an exception, if we caught `ValueError: Downloaded file size is not the same as expected` we could give users a message to raise an issue at https://github.com/andrey-yantsen/plexiglas/issues/new.

**Comment by worried-networking** [comment]
Yeah, good idea, I'll try to do some magic for all unhandled exceptions
